src/pages/components/tag/accessibility.mdx

```diff
@@ -19,205 +19,70 @@ import {
+in mind if you are modifying Carbon or creating a custom component.
+
+</PageDescription>
+
 <AnchorLinks>
-  <AnchorLink>How it works</AnchorLink>
-  <AnchorLink>Accessibility considerations</AnchorLink>
-  <AnchorLink>Resources</AnchorLink>
-  <AnchorLink>Accessibility testing</AnchorLink>
+  <AnchorLink>What Carbon provides</AnchorLink>
+  <AnchorLink>Development considerations</AnchorLink>
 </AnchorLinks>
 
 <A11yStatus layout="table" components="Tag" />
 
-## How it works
+## What Carbon provides
 
-Tags are used for web content that needs to be labeled, categorized, or
-organized using keywords that describe them. Tags are often used as a filter
-where all tags are in the same color, or used when content is mapped to multiple
-categories, where color is used to differentiate between categories.
+Carbon bakes keyboard operation into its components, improving the experience of
+blind users and others who operate via the keyboard. Carbon incorporates many
+other accessibility considerations, some of which are described below.
 
-## Accessibility considerations
+### Keyboard interaction
 
-This component has been validated to meet the
-[WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) and
-[Section 508](http://www.section508.gov/) accessibility guidelines, however
-changes made by the content owner can affect accessibility compliance. Be sure
-to review and follow the guidance in this section when updating or adding new
-content to this component.
+Read-only tags are not in the tab order, are not interactive, and do not receive
+focus.
 
-1. Be sure the tag text is clear and concise.
-2. Color should not be used as the only means to differentiate tag categories.
-3. When using custom colors be sure the minimum contrast requirements are met.
-4. Tags that are modified as a link to filter content should also follow the
-   link Carbon Component guidance.
+Dismissible tags are in the tab order and receive focus around the close icon.
+Pressing `Enter` or `Space` will dismiss the tag. Tabbing away from the tag will
+move focus to the next element in the tab order.
 
-## Resources
+<Row>
+<Column colLg={8}>
 
-#### Helpful resources for creating customized accessible implementations
+![Dismissible tag in the tab order receiving focus on the close icon to potentially dismiss the tag.](images/tag-accessibility-dismissible.png)
 
-- [IBM Accessibility Requirements](https://www.ibm.com/able/requirements/requirements/):
-  - [1.4.1 Use of Color](https://www.ibm.com/able/requirements/requirements/#1_4_1)
-    (WCAG Success Criteria
-    [1.4.1](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color))
-  - [1.4.3 Contrast (Minimum)](https://www.ibm.com/able/requirements/requirements/#1_4_3)
-    (WCAG Success Criteria
-    [1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum))
+</Column>
+</Row>
 
-## Accessibility testing
+Focus is shown around each selectable tag. Pressing `Enter` or `Space` toggles
```

Comment:
I think this is missing a short intro to the selectable tag. For example: 
- Read-only...
- Dismissable tag...
- Selectable tag...
- Operational tag...

Could be as repetitive as the Dismissable tag, but I think it's helpful to know the tab order receiving focus on the entire tag button.

Reply:
I did have that before but removed it. I can add it back